package.json

```diff
@@ -26,24 +26,24 @@
     "test:unit:watch": "jest --watch",
     "test:unit:coverage": "jest --coverage",
     "test:visual": "yarn test:visual:install && yarn test:visual:chrome && yarn test:visual:firefox && yarn test:visual:mobile",
-    "test:visual:install": "yarn selenium-standalone install --silent",
-    "test:visual:chrome": "yarn codeceptjs --config codecept-visual.conf.js run --steps --grep @desktop",
-    "test:visual:firefox": "yarn codeceptjs --config codecept-visual.conf.js run --steps --profile firefox --grep @desktop",
-    "test:visual:mobile": "yarn codeceptjs --config codecept-visual.conf.js run --steps --grep @mobile",
-    "test:functional:debug": "yarn cypress open",
+    "test:visual:install": "selenium-standalone install --silent",
+    "test:visual:chrome": "codeceptjs --config codecept-visual.conf.js run --steps --grep @desktop",
+    "test:visual:firefox": "codeceptjs --config codecept-visual.conf.js run --steps --profile firefox --grep @desktop",
+    "test:visual:mobile": "codeceptjs --config codecept-visual.conf.js run --steps --grep @mobile",
+    "test:functional:debug": "cypress open",
     "test:functional": "yarn test:functional:chrome && yarn test:functional:firefox",
-    "test:functional:firefox": "yarn cypress run --browser firefox",
-    "test:functional:chrome": "yarn cypress run --browser chrome",
-    "prepublish": "yarn run lib",
+    "test:functional:firefox": "cypress run --headless --browser firefox",
+    "test:functional:chrome": "cypress run --headless --browser chrome",
```

Comment:
If you run `yarn cypress` it should be always taken from the `node_modules`. If you run just `cypress` and you happen to have cypress installed globally which one is used?

The reason I like to have `yarn run` is that it explicitly tells you are running a custom script from `package.json`. For example if you had a script named `version` you would not be able to run it using `yarn version` since there is a built-in yarn command like that.

Reply:
> which one is used

the local one is used, and if it's not in node_modules - then the globally installed one is used.

I personally don't like `run` for its explicitness; I'm a big fan of doing `yarn` instead of `yarn install`, `npm i` instead of `npm install`, etc. However, since this topic really smells like a holywar, I'll just leave these lines as I've found them :)

Reply:
> the local one is used
👍 

> smells like a holywar
I am fine with new ideas. I too write `yarn install` in terminal because its short. However the `package.json` is written just once 😃 I am fine with changing this if the rest of the team shares your view on this.
WDYT @Gotusso @lfons @antoniofull ?

Reply:
If you ask me we should favour readability over shortness because we will read it a thousand times and yada yada yada 🤷‍♂️ But yeah, holy war, and seems a bit out of topic for this PR.